app/jobs/create_civic_vcfs.rb

```diff
@@ -1,7 +1,13 @@
 class CreateCivicVcfs < ActiveJob::Base
-  def perform
+  attr_reader :recurring
+
+  after_perform do |job|
+    job.reschedule if job.recurring
+  end
+
+  def perform(recurring = true)
+    @recurring if recurring
```

Comment:
You'll want to assign `@recurring` the value of `recurring` here, this line doesn't do anything as written.

Reply:
Duh. I fixed it now.